### PR TITLE
Make Galar formes show up in /ds

### PR DESCRIPTION
**Pull request by ZardMX**

### CODE REVIEW

File: server/chat-plugins/datasearch.ts [hunk 1]
```diff
@@ -1032,9 +1032,9 @@ function runDexsearch(target: string, cmd: string, canAll: boolean, message: str
 	let results: string[] = [];
 	for (const mon of Object.keys(dex).sort()) {
 		if (singleTypeSearch !== null && (dex[mon].types.length === 1) !== singleTypeSearch) continue;
-		const isAlola = dex[mon].forme === "Alola" && dex[mon].name !== "Pikachu-Alola";
+		const isAlolaORGalar = (dex[mon].forme === "Galar" || dex[mon].forme === "Alola") && dex[mon].name !== "Pikachu-Alola";
```
Comment (AnnikaCodes) [inline server/chat-plugins/datasearch.ts:1035]:
```suggestion
		const isRegionalForm = (dex[mon].forme === "Galar" || dex[mon].forme === "Alola") && dex[mon].name !== "Pikachu-Alola";
```

File: server/chat-plugins/datasearch.ts [hunk 2]
```diff
@@ -1032,9 +1032,9 @@ function runDexsearch(target: string, cmd: string, canAll: boolean, message: str
...
 		const allowGmax = (gmaxSearch || tierSearch);
-		if (!isAlola && dex[mon].baseSpecies && results.includes(dex[mon].baseSpecies)) continue;
+		if (!isAlolaORGalar && dex[mon].baseSpecies && results.includes(dex[mon].baseSpecies)) continue;
```
Comment (AnnikaCodes) [inline server/chat-plugins/datasearch.ts:1037]:
```suggestion
		if (!isRegionalForm && dex[mon].baseSpecies && results.includes(dex[mon].baseSpecies)) continue;
```

**Review by AnnikaCodes — CHANGES_REQUESTED** [review]

**Comment by ZardMX** [comment]
bump?

**Comment by Zarel** [comment]
@AnnikaCodes do you approve this now?

**Review by AnnikaCodes — APPROVED** [review]
Looks fine to me.

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**